Detectors/TPC/base/src/TPCBaseLinkDef.h

```diff
@@ -21,6 +21,7 @@
 #pragma link C++ class o2::tpc::CalArray < unsigned> + ;
 #pragma link C++ class o2::tpc::CalArray < short> + ;
 #pragma link C++ class o2::tpc::CalArray < bool> + ;
+#pragma link C++ class o2::tpc::CalArray < o2::tpc::PadFlags> + ;
```

Comment:
@sawenzel for the record, as discussed in the meeting, adding this line makes the code systematically crash.